doc_user/UserManual.adoc

```diff
@@ -259,15 +261,17 @@ The OPD Extended module affords the following additional features:
 * the OPD number now is visible (2): it is the progressive number per year automatically set by the system, changes to the value can be made only at the first registration in a year, then it will be increased automatically at each registration. The OPD number *MUST* be unique for each year.
 * Registered Patient (3): before choosing a disease, first select a patient from the drop-down list (this is *required*); to find a patient use the search field nearby by typing part of the patient's name or code; this improves the reliability of reports and statistics, and therefore is a service to the patient by knowing his/her history.
 * Once the patient is selected and a previous OPD visit is found, special fields are populated with the most recent visit of the patient (4), his/her personal data (5), and the Re-Attendance checkbox will be selected automatically upon selection of the main (first) disease (6);
-* It is now possible to set the next appointment date (7). This automatically schedules a visit visible in *_Therapy Management_*.
-* if a new attendance for this patient is required set the "New Admittance" check box and proceed normally.
+* It is now possible to set the next appointment date (7) and in what ward (OPD or IPD). This automatically schedules a visit visible in <<therapy-management,Therapy Management>> and <<worksheet,Worksheet>>.
+* It is compulsory to select the OPD ward (8) where the visit is taking place. Nevertheless, after first insertion in the session, the application will remember the latest selected to ease the daily data entry.
```

Comment:
I would reword: `It is compulsory to select the OPD ward (8) where the visit is taking place. Nevertheless, after first insertion in the session, the application will remember the latest selected to ease the daily data entry.` as `The selection of the the OPD ward (8)  is required so as to specify where the visit is taking place. After the first selection in a session, the application remembers the selection to ease the data entry.`